src/js/utils/__tests__/ReducerUtil-test.js

```diff
@@ -0,0 +1,163 @@
+const ReducerUtil = require('../ReducerUtil');
+
+const idReducer = function (state = '', action) {
+  if (action.path.join('') === 'id') {
```

Comment:
Well for the test this is probably to specific. 

My take on the idea for the path being an array was that you also could deep traverse into an object with logic if you have a path. This would also work if you path string ist something like: `container.docker.id` which you would split to an array. But then fields are not allowed to have a . in the name.

I will change this here to be a `String`.